src/pages/AddItem/components/InputItem.jsx

```diff
@@ -0,0 +1,14 @@
+function InputItem({ id, label, value, onChange, placeholder, onKeyDown, isTextArea }) {
+  return (
+    <div>
+      {label && <label htmlFor={id}>{label}</label>}
+      {isTextArea ? (
+        <div id={id} value={value} onChange={onChange} placeholder={placeholder} />
+      ) : (
+        <div id={id} value={value} onChange={onChange} onKeyDown={onKeyDown} placeholder={placeholder} />
+      )}
```

Comment:
```suggestion
<div
  id={id}
  value={value}
  onChange={onChange}
  onKeyDown={isTextArea ? onKeyDown : undefined}
  placeholder={placeholder}
/>
```

다 똑같고 하나만다르면 JSX를 똑같은걸 표현하지않고 해당 prop에 직접 식을 만들 수 있어요.
중요한건 onKeyDown은 이벤트 핸들러를 받기때문에 null이 아닌 undefined로 해야 합니다.